core/templates/dev/head/services/SearchServiceSpec.ts

```diff
@@ -217,36 +217,6 @@ describe('Search service', function() {
     }
   );
 
-  it('should only find empty string when ampersand is not escaped anywhere',
```

Comment:
Given the description of this test, shouldn't we still keep it, perform the search, but ensure that only the relevant fields are sent to the search service?

Reply:
Yes, sounds good. I'll do that. Thanks!